### PR TITLE
visibility 🇵🇸 🖤

### PR DESCRIPTION
**Pull request by acao**

### CODE REVIEW

**Comment by illBeRoy** [comment]
guess that Israeli lives don't matter as much

**Comment by eranshapira** [comment]
> guess that Israeli lives don't matter as much

some things never change.

**Comment by yuvalapidot** [comment]
This PR was not reviewed by anyone, this should be a red flag for using the project. Peace and love

**Comment by adylevy** [comment]
Shame.

**Comment by Evilu** [comment]
> This PR was not reviewed by anyone, this should be a red flag for using the project. Peace and love

I think you've just got on the saddest part, This is the complete opposite of what open source is about.

**Review by marat-asadurian — COMMENTED** [review]
This is a call for action, but the instructions are not clear. 

- How are we going to free Palestine? 
- From what are we going to free Palestine?
- What is the scope of Palestine?
I would like to see a high-level design & architecture document before submitting a call for action.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!-- @format -->
+> # Black Lives Matter 🖤 Free Palestine 🇵🇸
```
Comment (eidan66) [inline README.md:2]: Remove `Free Palestine 🇵🇸`.

**Review by eidan66 — COMMENTED** [review]
Remove `Free Palestine 🇵🇸`.

**Comment by rebeccajfriedman** [comment]
Hey @acao - the comments on this PR and the few related open PRs speak pretty clearly. We need to ensure that the open source community is inclusive and politics free. 

I'd like to see this PR reverted.

cc @imolorhe @yoshiakis
cc @leebyron @asiandrummer @wincent @lostplan @IvanGoncharov

**Comment by semeonb** [comment]
Revert Palestine to ver 1.0 Cna'an.

**Comment by akaterra** [comment]
@acao please revert this commit. As israeli it is unpleasant for me to see such non-clarifying calls that can be interpreted one-sidedly as it is convenient for each one. I believed that github is a policy-free place and I find it strange to see it in one of most popular open source projects.

**Comment by leonzinger** [comment]
> _No description provided._

 I guess this is what happens when you feed yourself from biased media. Shame on you for involving politics in here. We can have lots of discussions on politics but this is not the place for this!

**Comment by leebyron** [comment]
Thank you to everyone who raised concerns in a civil manner.

The GraphQL technical steering committee is working on a policy to guide the use of political speech from the voice of a GraphQL project. The original author chose to revert this change via #1879 until we complete that policy.

As a reminder, all discussion within GraphQL projects must adhere to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md).

**Pull request merged**